src/assets/styles/_base.scss

```diff
@@ -8,8 +8,9 @@ body {
   margin: 0;
   tab-size: 2;
   color: var(--black-500);
-  font-size: 1em;
   font-family: var(--font-family-sans);
+  font-weight: var(--font-weight-normal, 400);
```

Comment:
**Suggestion**: This should be `--font-weight-regular` from `src/assets/styles/_variables.scss`.

```suggestion
  font-weight: var(--font-weight-regular, 400);
```

Reply:
This was set to `--font-weight-normal` on purpose, at the moment it fallsback to `400` as we don't currently use `-normal`.

I want to try to align to more pseudo-standard things so that things "Just Work", see another project below:

![Screenshot 2023-06-01 at 10 12 44](https://github.com/kumahq/kuma-gui/assets/554604/feacc159-3fad-439c-a908-4922917c6910)

Reply:
We have those variables specifically for that purpose. Changing it should also affect the font-weight here.

Why should it be a new, different variable?

Reply:
Because this is what they do in Konnect, I would like our CSS to be as close as possible to Konnect and avoid overwriting things as much as possible.

I think if anything we should change our existing variable name to be exactly the same as Konnect/Kongponents. 

Honestly this one isn't doing anything at the moment, but the point is, if the variable gets changed by the Konnect team in Konnect, then our will just automatically update to use the same setting without us doing anything, and that's the main point.